auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -316,11 +316,15 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
 
     this.isLoading = 1 // truthy
     this.$route.query.isNumberedCompanyRequest && await this.createNumberedBusiness(this.currentAccountSettings.id)
```

Comment:
Does `createNumberedBusiness()` still exist, or replaced by `createDraftFiling()` ?

See also line 262.

Reply:
`createNumberedBusiness` still exist in store. I replaced it in service